deploy/buildtools/buildafi.py

```diff
@@ -191,7 +261,7 @@ def aws_create_afi(global_build_config, buildconfig):
     # append the build node IP + a random string to diff them in s3
     global_append = "-" + str(env.host_string) + "-" + ''.join(random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(10)) + ".tar"
 
-    with lcd("""{}/results-build/{}/cl_firesim/build/checkpoints/to_aws/""".format(ddir, builddir)), StreamLogger('stdout'), StreamLogger('stderr'):
+    with lcd("""{}/cl_{}/build/checkpoints/to_aws/""".format(local_results_dir, tag_buildtriplet)), StreamLogger('stdout'), StreamLogger('stderr'):
```

Comment:
Correct. By default all builds happen in a `cl_TRIPLET` folder instead of `cl_firesim`. In the local case, this is still in the `aws-fpga` area directly found in current FireSim repo.

Reply:
When you copy the cl_dir into local_results_dir at the end, are you copying it into `f"{local_results_dir}/cl_firesim"` or `f"{local_results_dir}/cl_TRIPLET"`?  I only ask because I don't see that copy showing up in the changes.

Reply:
I am copying back into the `cl_<TRIPLET>` area.

This is where the CL_DIR name is determined - https://github.com/firesim/firesim/blob/417ebe654d8ee6c71c18cd473bd0e4dc2632352b/deploy/buildtools/buildafi.py#L119
This is where the CL_DIR is copied back (with the same name) - https://github.com/firesim/firesim/blob/417ebe654d8ee6c71c18cd473bd0e4dc2632352b/deploy/buildtools/buildafi.py#L201-L208